src/Pages/Uptime/BulkImport/Upload.jsx

```
@@ -0,0 +1,83 @@
+import { useTheme } from "@emotion/react";
+import { useState, useRef, useEffect } from "react";
+import { Button, Typography } from "@mui/material";
+import { parse } from "papaparse";
+import { bulkMonitorsValidation } from "../../../Validation/validation";
+
+export function Upload({ onComplete }) {
```

ESLint

[error] 'onComplete' is missing in props validation

(react/prop-types)